.gitignore

```diff
@@ -7,6 +7,7 @@ yarn.lock
 coverage
 
 .idea/
+.vs
```

Comment:
What is this exclude for?

Reply:
My vscode creates this file that should not be commited. See https://stackoverflow.com/questions/31878901/should-i-add-the-visual-studio-2015-vs-folder-to-source-control